### PR TITLE
:sparkles: menu

### PR DESCRIPTION
**Pull request by fabienjuif**

### CODE REVIEW

**Comment by fabienjuif** [comment]
@guillaumecrespel ready for review

File: styles/_colors.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 $primary-color: #50e39e !default;
 $primary-color-lighten-1: lighten($primary-color, 7%) !default;
 $primary-color-darken-1: darken($primary-color, 7%) !default;
+$primary-color-darken-2: darken($primary-color, 12%) !default;
```
Comment (guillaumecrespel) [inline styles/_colors.scss]: the color isn't define by percent,
prefer this notation => $primary-color-darken-12: darken($primary-color, 12%) !default;  
?

Comment (fabienjuif) [inline styles/_colors.scss]: I did this on purpose, so we can change % without having breaking change.

1 -> ligther
2 -> even ligher

or

1 -> darker
2 -> even more darker

Comment (guillaumecrespel) [inline styles/_colors.scss]: :+1: I think is a good compromise

**Pull request merged**